lib/solr_configurator.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require 'zip'
+
+class SolrConfigurator
+  def initialize; end
```

Comment:
You can actually remove this empty initialize method entirely, it'll be inherited from `Object`. It's fine to leave it here, it won't do any harm, but it's not necessary and the Ruby idiom would be to remove it